### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by haolin512900** [microsoft/Bringing-Old-Photos-Back-to-Life#157]
Dear  Wan，
       I'm a postgraduate student of Yunnan University in China.
       l just took my first step in scientific research focused on data mining and recommendation.Recently I read your paper
 I think it's very interesting and your clear and compact declaration inspired me a lot. But I got some trouble when I try to implement the algorithms.
    So could you please send me  the scratched old photos dataset with labels directly of the paper if possible? 1 would be very appreciated if you can help me : )
    Thank you very much !
    Best wishes.
Yours respectfully.

**Comment by hello-trouble** [comment]
Hello, I am also puzzled about the dataset. How to get the dataset used in the training of this paper? Do you solve it now?

**Comment by shumile66** [comment]
> Dear Wan，
> I'm a postgraduate student of Yunnan University in China.
> l just took my first step in scientific research focused on data mining and recommendation.Recently I read your paper
> I think it's very interesting and your clear and compact declaration inspired me a lot. But I got some trouble when I try to implement the algorithms.
> So could you please send me the scratched old photos dataset with labels directly of the paper if possible? 1 would be very appreciated if you can help me : )
> Thank you very much !
> Best wishes.
> Yours respectfully.

May I ask, has your problem been solved?

**Comment by shumile66** [comment]
> Hello, haolin512900 . I am also puzzled about the dataset. How to get the dataset used in Create_Bigfile.py? such as 'Real_L_old'and 'Real_RGB_old'. Looking forword your help and thank you in advance.
> 
> > Dear Wan，
> > I'm a postgraduate student of Yunnan University in China.
> > l just took my first step in scientific research focused on data mining and recommendation.Recently I read your paper
> > I think it's very interesting and your clear and compact declaration inspired me a lot. But I got some trouble when I try to implement the algorithms.
> > So could you please send me the scratched old photos dataset with labels directly of the paper if possible? 1 would be very appreciated if you can help me : )
> > Thank you very much !
> > Best wishes.
> > Yours respectfully.
> 
> Hello, I am also puzzled about the dataset. How to get the dataset used in Create_Bigfile.py? such as 'Real_L_old'and 'Real_RGB_old'. Looking forword to your help and thank you in advance.

May I ask, has your problem been solved?Which version of pyTorch are you using?
